code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -85,3 +87,53 @@
 	head = null
 	mask = /obj/item/clothing/mask/gas/sechailer
 	internals_slot = ITEM_SLOT_SUITSTORE
+
+//TEST THIS OUT WHEN NEEDED
+
+/obj/item/choice_beacon/head_of_security
+	name = "gun choice beacon"
+	desc = "whatever you choose will determine the outcome of space station 13 and the fate of the company so choose wisely."
```

Comment:
Should probably be "Whatever you choose will determine the outcome of space station 13 and the fate of the company. Choose wisely."

Also, "the fate of the company" sounds very pretentious. Maybe instead of "will determine" "may well determine"